### ISSUE TITLE
Emoji in widget text causes potential crash

### ISSUE THREAD

**Issue by vicnguyenqt** [NodeBB/nodebb-widget-essentials#63]
Hi,
Yesterday we did a Nodebb maintenance from v1.x to 2.8.0. Everything went fine.
But at the end of the last minute, we ended up with a serious bug and after a few mins of investigating I'm hereby to report it

### Version:
node: 19.6
npm: 6.14.18
plugin: "nodebb-widget-essentials": "4.1.2"
nodebb: v1.x

### How the bug looks like
One of our admins of NodeBB added a maintenance message with emoji. Everything worked fine, but when I tried to remove the message from the banner, got the error message and crash the whole app (see below):
`Feb 06 15:37:56  [app/web.3] panicked at 'byte index 4 is not a char boundary; it is inside '⚠' (bytes 3..6) of `<p>⚠️ Forum Maintenance: Feb 6th, 8am - 14pm (UTC+2)</p>-->`
![MicrosoftTeams-image (9)](https://user-images.githubusercontent.com/93513978/217252933-dafa3393-d6fc-4ec7-97c3-0fd28d4d98d5.png)

After investigated, we need to do a dirty hack to remove the message from MongoDB database. Eventually, the app alive again, and diagnosed that this came from the plugin "nodebb-widget-essentials"

Please have a look at it. Let me know if you guys need more information

**Comment by barisusakli** [comment]
I was able to successfully add a html widget with the content `⚠️ Forum Maintenance: Feb 6th, 8am - 14pm (UTC+2)` and remove it without issue on nodebb 2.x. 

The stack trace points to the compiler in benchpressjs compiler.

```
module.exports.__wbg_new_59cb74e423758ede = function() {
    var ret = new Error();
    return addHeapObject(ret);
};
```
You can try with nodejs 16 / 18, we don't support nodejs 19.

Can you reproduce it when you re add the same html widget on a dev instance?

**Comment by pitaj** [comment]
I was able to reproduce with the following template:

```html
<!--<p>⚠️ Forum Maintenance: Feb 6th, 8am - 14pm (UTC+2)</p>-->

```

**Comment by pitaj** [comment]
Fixed by benchpressjs@2.5.0
